runconfig/opts/parse.go

```diff
@@ -473,7 +473,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 	return config, hostConfig, networkingConfig, cmd, nil
 }
 
-// reads a file of line terminated key=value pairs and override that with override parameter
+// reads a file of line terminated key=value pairs and overrides that with override parameter
```

Comment:
"reads a file of line terminated key=value pairs, and overrides any keys present in the file with additional pairs specified in the override parameter"?